roles/kubernetes-control-plane/tasks/main.yml

```
@@ -1,3 +1,175 @@
+- name: Pre configure control plane hooks
+  ansible.builtin.include_tasks: "{{ item }}"
+  with_items: "{{ kubernetes_hookfiles.pre_configure_control_planes }}"
+  when:
+  - kubernetes_hookfiles.pre_configure_control_planes is defined
```

Change item to something unique to avoid warning/duplicating use of item.
```suggestion
- name: Pre configure control plane hooks
  ansible.builtin.include_tasks: "{{ pre_configure_control_plane_hook}}"
  loop: "{{ kubernetes_hookfiles.pre_configure_control_planes }}"
  loop_control:
    loop_var: pre_configure_control_plane_hook
  when:
  - kubernetes_hookfiles.pre_configure_control_planes is defined
```